packages/x-components/src/components/panels/base-tabs-panel.vue

```diff
@@ -0,0 +1,331 @@
+<template>
+  <section v-if="getTabs().length > 0" class="x-tabs-panel" data-test="base-tabs-panel">
+    <component
+      :is="animation"
+      tag="ul"
+      class="x-list"
+      :class="tabsListClass"
+      data-test="base-tabs-panel-list"
+    >
+      <li v-for="tab in getTabs()" :key="tab">
+        <!--
+          @slot Slot that is be used for replacing the whole tab.
+            @binding {tab} string - The tab name.
+            @binding {isSelected} boolean - Indicates if the tab is selected.
+            @binding {selectTab} {() => void} selectTab - Function to select the tab.
+         -->
+        <slot
+          name="tab"
+          v-bind="{ tab, isSelected: tabIsSelected(tab), selectTab: () => selectTab(tab) }"
+        >
+          <button
+            :key="tab"
+            @click="selectTab(tab)"
+            class="x-button x-tabs-panel__button"
+            :class="tabIsSelected(tab) ? activeTabClass : tabClass"
+            :aria-pressed="tabIsSelected(tab)"
+            data-test="base-tabs-panel-button"
+          >
+            <!--
+              @slot Slot used to just pass the content.
+                @binding {tab} string - The tab name.
+                @binding {isSelected} boolean - Indicates if the tab is selected.
+            -->
+            <slot name="tab-content" v-bind="{ tab, isSelected: tabIsSelected(tab) }">
+              {{ tab }}
+            </slot>
+          </button>
+        </slot>
+      </li>
+    </component>
+
+    <div
+      v-if="selectedTab && $scopedSlots[selectedTab]"
+      :key="selectedTab"
+      :class="panelClass"
+      data-test="base-tabs-panel-content"
```

Comment:
I find these twoo lines a little bit confusing as we are naming as "panel" this element when talking about classes, but as "content" when talking about tests. As the component itself is named "tabs panel", I'd go with "content" always.
```suggestion
      :class="contentClass"
      data-test="base-tabs-panel-content"
```

Reply:
Sounds good to me, I'll change it, thanks!